packages/components/scripts/jobs/vue/createWrappers.js

```diff
@@ -306,6 +311,10 @@ defineEmits<{
     ${slots}
   </${component.tagName}>
 </template>
+
+<script lang="ts">
```

Comment:
Would it be possible to prevent empty `<script lang="ts" />` by checking if the exports map has values, something like?

```typescript
const eventExports = getEventExports(component.events);
const exports = eventExports.length > 0 ? `<script lang="ts">${eventExports}</script>` : '';
```